includes/rest/class-server.php

```diff
@@ -108,8 +108,28 @@ public static function authorize_activitypub_requests( $response, $handler, $req
 
 		// POST-Requets are always signed
 		if ( 'GET' !== $request->get_method() ) {
+
+			if ( Application::is_actor_delete_request( $request ) ) {
+				if ( ! Application::is_known_commenter( $request ) ) {
```

Comment:
I would not handle that here, because this skips all the inbox actions, so no other implementation could interpret and react to it. lets skip signature verification for `Deletes` (even with the introduced filter or directly in the server class) and implement the code in the delete handler I am introducing here: https://github.com/Automattic/wordpress-activitypub/pull/551

I would also prefer adding the delete code to the matching collections, like followers for the actor deletes and https://github.com/Automattic/wordpress-activitypub/pull/551/files#diff-17683f8d8c7226d3e07a6c94ffb52b7bf270d9711a36bdb118b6b4c55fd32820 (we can discuss the name of it) for the comment/reaction deletes